src/docker/processor/start.sh

```diff
@@ -13,4 +13,23 @@ if [[ "$HEALTHCHECK_BEFORE_START" == "true" ]];then
     done
 fi
 
-/usr/local/bin/processor -c /config/data/config.yaml
+psql $DATABASE_URL -c '\copy processor_status to out.csv csv'
+
+if [ -s "out.csv" ];then
+    export STARTING_VERSION=$(cut -d, -f2 out.csv)
+fi
```

Comment:
When trying to deploy on cloud I've noticed issues when storing in root dir

How about storing at a `/app/status_at_startup.csv`?